time stamp

# constants for emotions
int32 SAD=1
int32 ANGRY=2
int32 DISGUSTED=3
int32 HAPPY=4
int32 SURPRISE=5
int32 SHY=6

# constants for feedback
int32 ENCOURAGE=1
int32 CHALLENGE=2

int32 id
float32[] emotions
int32 difficulty_level #length of emotions
int32 feedback

bool perform_feedback